app/models/viewModels/ChangeSuccessViewModel.scala

```diff
@@ -18,5 +18,4 @@ package models.viewModels
 
 case class ChangeSuccessViewModel(pageTitle: String,
                                   agentEmail: Option[String],
-                                  businessName: Option[String],
-                                  contactPreference: Option[String])
+                                  businessName: Option[String])
```

Comment:
Neither of the success views use `agentEmail` or `businessName`. As far as I can tell these items can be removed, and considering this model would then be only one parameter, I would say you can remove this entirely and simply pass in the dynamic trading name title to that view.